Makefile

```diff
@@ -17,8 +17,8 @@ GOOSES = darwin linux
 NOTARY_BUILDTAGS ?= pkcs11
 NOTARYDIR := /go/src/github.com/docker/notary
 
-GO_VERSION := $(shell go version | grep "1\.[6-9]\(\.[0-9]+\)*")
-# check to make sure we have the right version
+GO_VERSION := $(shell go version | grep "1\.[6-9]\(\.[0-9]+\)*\|devel")
+# check to make sure we have the right version. developers are on their own.
```

Comment:
Apologies for the nitpick, but would you mind clarifying this statement to say something like "development versions of Go are not officially supported, but allowed for building"?